parquet/src/test/scala/magnolify/parquet/test/util/AvroSchemaComparer.scala

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2022 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package magnolify.parquet.test.util
+
+import org.apache.avro.Schema
+import cats.implicits._
+import scala.jdk.CollectionConverters._
+
+trait AvroSchemaComparer {
```

Comment:
This is too complex. We should validate and acknowledge expected diffs if any

Reply:
Agreed